packages/obuilder/obuilder.0.5/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "Run build scripts for CI"
+description:
+  "OBuilder takes a build script (similar to a Dockerfile) and performs the steps in it in a sandboxed environment."
+maintainer: ["talex5@gmail.com"]
+authors: ["talex5@gmail.com"]
+license: "Apache-2.0"
+homepage: "https://github.com/ocurrent/obuilder"
+doc: "https://ocurrent.github.io/obuilder/"
+bug-reports: "https://github.com/ocurrent/obuilder/issues"
+depends: [
+  "dune" {>= "3.3"}
+  "lwt" {>= "5.6.1"}
+  "astring"
+  "fmt" {>= "0.8.9"}
+  "logs"
+  "cmdliner" {>= "1.1.0"}
+  "tar-unix" {>= "2.0.0"}
+  "yojson" {>= "1.6.0"}
+  "sexplib"
+  "ppx_deriving"
+  "ppx_sexp_conv"
+  "sha" {>= "1.15.1"}
+  "sqlite3"
+  "obuilder-spec" {= version}
+  "ocaml" {>= "4.14.0"}
+  "alcotest-lwt" {with-test}
+  "odoc" {with-doc}
```

Comment:
```suggestion
  "alcotest-lwt" {with-test}
  "odoc" {with-doc}
  "camlp-streams"
```
This is required to make stream work with `OCaml 5`

Reply:
No, we're not using `Stream` in obuilder, the error comes from [tar](https://github.com/mirage/ocaml-tar).

Reply:
Indeed! My mistake :sweat_smile: